src/index.ts

```diff
@@ -1,3 +1,5 @@
 import { downloadRelease } from './downloadRelease';
+import { HTTPError } from './getReleases';
 
 export { downloadRelease };
+export { HTTPError };
```

Comment:
this is just consistency really but usually we either export each individual item, or export a single object with what we want exposed. I don't think we need to export double objects